### ISSUE TITLE
Queuable with ShouldBeEncrypted error!!

### ISSUE THREAD

**Issue by AnassSamaa** [tenancy/tenancy#276]
Using ShouldBeEncrypted on a queuable object throws

unserialize error in  tenancy/identification-driver-queue/Events/Processing.php:60

**Comment by ArlonAntonius** [comment]
Hi @AnassSamaa,

Any chance you can provide a stack trace so we can do some debugging on this? We would really want this to work properly.

**Comment by makroxyz** [comment]
@ArlonAntonius just try to add ShouldBeEncrypted to a Queuable

```php
class Activity implements ShouldBeEncrypted, ShouldQueue
{
    use Dispatchable;
    use InteractsWithQueue;
    use Queueable;
    use SerializesModels;

    ....
```

**Comment by ArlonAntonius** [comment]
Fixed in #287

**Comment by ArlonAntonius** [comment]
Tagged v2.2.0 which includes a fix for this 🚀
